package.json

```diff
@@ -92,6 +92,18 @@
                     "scope": "resource",
                     "type": "array"
                 },
+                "black-formatter.showNotification": {
+                    "default": "on-warn",
+                    "description": "When extension is allowed to notifications messages in UI.",
+                    "enum": [
+                        "on-crash",
+                        "on-error",
+                        "on-warn",
+                        "on-message"
```

Comment:
I feel like these should be `on-error`, `on-warn`, `always`, and `never`. I also recommend adding enum descriptions.

An example, for enum descriptions:
```
                "python.languageServer": {
                    "default": "Default",
                    "description": "Defines type of the language server.",
                    "enum": [
                        "Default",
                        "Jedi",
                        "Pylance",
                        "None"
                    ],
                    "enumDescriptions": [
                        "Automatically select a language server: Pylance if installed and available, otherwise fallback to Jedi.",
                        "Use Jedi behind the Language Server Protocol (LSP) as a language server.",
                        "Use Pylance as a language server.",
                        "Disable language server capabilities."
                    ],
                    "scope": "window",
                    "type": "string"
                },
```

/cc @luabud Need your input on the enum values and the descriptions.